cmd/internal/cmderr/checks.go

```diff
@@ -19,9 +22,37 @@ func RequirePostmanAPICredentials(explanation string) (string, error) {
 		} else {
 			printer.Infof("Please set the POSTMAN_API_KEY environment variable, either in your shell session or prepend it to postman-insights-agent command.\n")
 		}
-		//lint:ignore ST1005 This is a user-facing error message
 		return "", AkitaErr{Err: errors.New("Could not find a Postman API key to use")}
 	}
 
 	return key, nil
 }
+
+// Checks that an API key and a project ID are provided, and that the API key is
+// valid for the project ID.
+func CheckAPIKeyAndInsightsProjectID(projectID string) error {
+	// Check for API key.
+	_, err := RequirePostmanAPICredentials("The Postman Insights Agent must have an API key in order to capture traces.")
+	if err != nil {
+		return err
+	}
+
+	// Check that project ID is provided.
+	if projectID == "" {
+		return errors.New("project ID is missiing, it must be specified")
+	}
+
+	frontClient := rest.NewFrontClient(rest.Domain, telemetry.GetClientID())
+	var serviceID akid.ServiceID
+	err = akid.ParseIDAs(projectID, &serviceID)
+	if err != nil {
+		return errors.Wrap(err, "failed to parse service ID")
```

Comment:
Here also, an error message being shown to the user should be clear and actionable.

We shouldn't call it a 'service ID' in a user-facing message if it is referred to as a 'project ID' everywhere else.